Codeinwp/neve
# Show post meta elements in block editor for cover layout as default

### Summary

* If layout is "cover" mode for Single Post from customizer; the toggles for **Author Avatar** and **Reading Time** were not be visible in Post Edit Screen -> Neve Options until reset button is clicked. That was fixed with this PR. #3916 - screencast (before/after): https://www.loom.com/share/006de7b8432647b6951d40bdff8469dc
* The "Estimated Reading Time" in the Customizer->Layout->Single Post->Post Meta was not reflecting the individual Post Options (WP Admin->Post Edit->Neve Options), that's fixed with this PR. Now individual Reading Time enabling toggle gets inherit from Global customizer option https://www.loom.com/share/e7adeb0ca07d458882f3d0920020d7f2
* Code improvement related single post options default values overall ([get_default_single_post_meta_fields](https://github.com/Codeinwp/neve/blob/d5489d11235821a54dd9fe665cf7a73881bef932/inc/customizer/defaults/single_post.php#L225-L243) was created and that is used in the codebase)

### Will affect visual aspect of the product

YES

### Screenshots 

### Test instructions

- Create environment and install Neve
- Install Neve Pro from https://github.com/Codeinwp/neve-pro-addon/pull/2431 (individual Reading Time meta is not visible in stable version, it was fixed with this PR. Therefore, I suggest use this PR build for Neve Pro, note [the PR in Neve Pro side](https://github.com/Codeinwp/neve-pro-addon/pull/2431) was already tested and currently it's ready to merge.)
- Visit Customizer -> Layout -> Single Post and choose Header Layout as Cover
- Play with meta visibility & order settings for Single post in customizer and observe the behavior on front end.
- Override meta settings for a post on Post Edit screen and observe behaviors on the frontend.

**Cases:**
* Enabling/disabling the visibility of post meta (reading time, author avatar) etc. should work well as globally for the both header layout type (Cover/Normal)
* Enabling/disabling the visibility of post meta (reading time, author avatar) etc. should work well as individually (post level) for the both header layout type (Cover/Normal)
* No regression should occurs default values of the Neve Single Post settings overall in customizer and in Block Editor (WP Admin->Post Edit->Neve Options)

## Check before Pull Request is ready:



Closes #3916, #3932.
